src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -674,4 +674,17 @@ setx PATH "%PATH%;{0}"
   <data name="CommandPrereleaseOptionDescription" xml:space="preserve">
     <value>Allows prerelease packages to be installed.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SelfContainedOptionDescription" xml:space="preserve">
+    <value>Publish the .NET runtime with your application so the runtime doesn't need to be installed on the target machine.
+The default is 'true' if a runtime identifier is specified.</value>
+  </data>
+  <data name="NoSelfContainedOptionDescription" xml:space="preserve">
+    <value>Publish your application as a framework dependent application without the .NET runtime. A supported .NET runtime must be installed to run your application.</value>
+  </data>
+  <data name="SelfContainAndNoSelfContainedConflict" xml:space="preserve">
+    <value>The '--self-contained' and '--no-self-contained' options conflict with each other. Specify only one of the options.</value>
+  </data>
+  <data name="RuntimeOptionShouldBeUsedWithSelfContained" xml:space="preserve">
```

Comment:
This property has the opposite polarity it should have, I believe. I think it should be: `SelfContainedOptionShouldBeUsedWithRuntime`.